app.py

```diff
@@ -88,5 +104,28 @@
     health_check_interval=5,
 )
 
-# Generate stacks
+app_service_props_otel_collector = ServiceProps(
+    container_name="otel-collector",
+    container_port=4318,
+    container_memory=512,
+    container_location="ghcr.io/sage-bionetworks/sage-otel-collector:v0.0.4",
+    container_env_vars={},
+    container_secrets=[
+        ServiceSecret(
+            secret_name=f"{stack_name_prefix}-DockerFargateStack/{environment}/opentelemetry-collector-configuration",
+            environment_key="CONFIG_CONTENT",
```

Comment:
i don't see this key in the `schematic-dev-sandbox-DockerFargateStack/dev-sandbox/opentelemetry-collector-configuration` secret?

Reply:
`environment_key` means that the secret is going to be available as in the container.

Not a reference to any values in aws secret manager